services/discordService.js

```diff
@@ -51,8 +51,25 @@ const addRoleToUser = async (userid, roleid) => {
   return response;
 };
 
+const removeRoleFromUser = async (roleId, discordId) => {
```

Comment:
Can you please point out the place where we are using this function?

Reply:
We are not using it now will be using it in subsequent PRs as this pr just creates a service we will be using it at multiple places in near future